contracts/token/ERC721/extensions/ERC721Pausable.sol

```diff
@@ -35,6 +35,6 @@ abstract contract ERC721Pausable is ERC721, Pausable {
     ) internal virtual override {
         super._beforeTokenTransfer(from, to, firstTokenId, batchSize);
 
-        require(!paused(), "ERC721Pausable: token transfer while paused");
+        _requireNotPaused();
```

Comment:
Any reason why `ERC20Pausable` and `ERC1155Pausable` use `whenNotPaused` but not this ?


IMO we shoud do
```solidity
    function _beforeTokenTransfer(
        address from,
        address to,
        uint256 firstTokenId,
        uint256 batchSize
    ) internal virtual override whenNotPaused() {
        super._beforeTokenTransfer(from, to, firstTokenId, batchSize);
    }
```

Reply:
Flagged and solved [here](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4261#discussion_r1212762392)